WL4EditorWindow.cpp

```diff
@@ -1068,28 +1079,41 @@ bool WL4EditorWindow::UnsavedChangesPrompt(QString str)
 /// Clear eventhing in the current room.
 /// But at least one door will be kept.
 /// </summary>
-void WL4EditorWindow::CurrentRoomClearEverything()
+/// <param name="no_warning">
+/// Optional param for showing warning of deleting doors.
+/// </param>
+/// <param name="roomId">
+/// Optional param for selecting a room to clear, set -1 as a default value for current room.
+/// </param>
+void WL4EditorWindow::ClearEverythingInRoom(bool no_warning, int roomId)
```

Comment:
Is roomId used?

Reply:
Oh, i forget to use it. Fixed.

Reply:
roomId is not needed here. The only usage of roomId here is replacing selectedRoom, and the only time that ClearEverythingInRoom(bool, int) is called is after setting selected room to this value.